roadmap.md

```diff
@@ -1,145 +1,45 @@
 # .NET Core Roadmap
 
-We aim to continuously deliver updates to the .NET Core Runtime and Tools.
-We are primarily focused on delivering UWP 6.0 (.NET Standard 2.0 for UWP) and .NET Core 2.1.
+The .NET Core roadmap communicates project priorities for evolving and extending the scope of the product. New product experiences and features will include changes in various [layers of the product](Documentation/core-repos.md), in some combination of the runtime, framework, language compilers and tools. Each component may have its own component-level roadmap that will available in the repo for that component.
 
-We shipped .NET Core 2.0 and .NET Standard 2.0 - you can try it by installing [Visual Studio 2017 (15.3)][vs2017] and [.NET Core 2.0 SDK][Core20Sdk].
+The .NET Core team is currently focussed on Web and console apps. We encourage the community to work with us to improve web and console apps and to extend .NET Core for other scenarios.
```

Comment:
one s in focused. Does web cover cloud, containers, microservices, etc? When I read web, I think websites while we are definitely focused on more.